content/react-native/expo-tutorial-project/_index.md

```diff
@@ -13,55 +13,30 @@ Alrighty, time to get your hands dirty.
 
 Start off by doing [this tutorial](https://docs.expo.dev/tutorial/planning/). Make sure that you understand what you are doing, we'll be doing much harder stuff soon.
 
-Please be sure to include a `.nvmrc` file in your project submission. If you don't know what that means then go read about NVM.
-
-## Moar requirements
+## More requirements
 
 Once you have finished the basic tutorial, improve it by doing the following:
 
-1. Refactor your code so that each component is in a separate file
-2. Add a new button to clear the selected image state
-3. Create a reusable Button component to clean up duplication of TouchableOpacity / Text
+1. Include a `.nvmrc` file in your project submission. If you don't know what that means then go read about NVM.
+2. Refactor your code so that each component is in a separate file.
+3. Add a new button to clear the selected image and the selected emoji sticker.
 4. Make sure that you use `const` wherever appropriate. The tutorial uses `let` for everything, it's not always the best option. Most things are actually constant.
 
 ## Common problems
 
-### Section: Sharing the image
-
-If you get the error: `Uncaught (in promise) ReferenceError: Platform is not defined` then double check that your imports include `Platform`. Eg:
```

Comment:
Why were these instructions removed?

Reply:
The tutorial project has changed and there is not `Sharing the image` section.

I mentioned it on the PR description:
![image](https://github.com/Umuzi-org/ACN-syllabus/assets/54069197/4d0637c1-e6ab-4126-bdf1-be4d64ecd1a9)